BunnyHopMaster/Assets/Scripts/UI/PlayerUI.cs

```diff
@@ -42,6 +46,10 @@ public class PlayerUI : MonoBehaviour {
                 Pause();
             }
         }
+        if (Input.GetKeyDown(HotKeyManager.Instance.GetKeyFor(PlayerConstants.NextLevel)))
```

Comment:
add this to WinMenu.cs instead so that way they can't go to the next level without beating the current one!